### PR TITLE
Added route for individual shows

### PR DESCRIPTION
**Pull request by mathewantony31**
Wow, I went down a rabbit hole for this one. Here's what I'm thinking:
- Create a new route that displays a full page preview of an individual show
- When you share an individual show, we copy a direct link to _just_ that show. Friends who access that link will see an option to fetch their own schedule.

Ideally it'd look like this (if say you went to sxdiscover.com/show/showId):
screenshot 2019-02-12 21 30 26

With this diff, it currently looks like this (there were some complications fetching album art from Spotify but that's for a future diff):
screenshot 2019-02-12 21 30 19

### CODE REVIEW

File: routes/show.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        try{
+          console.log(docs)
+          var results = JSON.stringify(docs)
+          console.log(results);
+          var name = docs[0].name
+          var venue = docs[0].venue
+          var date = docs[0].date
+          var time = docs[0].time
+
+          res.render('show', {name:name, venue: venue, date:date, time:time})
```
Comment (benhummel) [inline routes/show.js]: nit: extra space after `venue`

File: views/show.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ga('send', 'pageview');
+  </script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+
+</head>
+<body>
+  <div class="container">
+    <div class="container__content">
+      <div class="container__content__band-name">{{name}}</div>
+      <div class="container__content__show-info">is playing {{venue}} on {{date}} at {{time}}.</div>
```
Comment (benhummel) [inline views/show.html]: for some reason in your screenshot it didn't show the date. is that happening in prod?

File: views/show.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+  </div>
+  <div id="view-your-schedule" class="view-your-schedule">
+    <div class="view-your-schedule__button-text">GET YOUR OWN SCHEDULE</div>
+  </div>
+  <script>
```
Comment (benhummel) [inline views/show.html]: copy ideas:
1.  "Get your own custom SX schedule"
2. Want to see your own custom SXSW schedule? **Connect with Spotify**

File: views/summary.html [hunk 4]
```diff
@@ -20,50 +20,9 @@
 <body>
   <div class="container">
 
-    <!-- if top or saved, show those -->
```
Comment (benhummel) [inline views/summary.html]: Did you mean to remove all this?

File: routes/show.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+// Route that's called to access a full page view for a specific show
+
+var express = require('express');
+var router = express.Router();
+var Band = require('../models/band.js');
+
+router.get('/show/*', function(req, res, next){
+  try{
+    console.log("Show ID is "+req.params[0])
```
Comment (benhummel) [inline routes/show.js]: can remove

File: routes/index.js [hunk 6]
```diff
@@ -29,6 +30,9 @@ router.get('/privacy-policy', function(req, res, next){
   console.log("Fetching privacy page. Current memory usage is "+(Math.round(process.memoryUsage().heapUsed/1048576))+" MB.")
```
Comment (benhummel) [inline routes/index.js]: remove

**Comment by benhummel** [comment]
Left some comments; feel free to fix and merge

File: views/show.html (hunk 2 above)
Comment (mathewantony31) [reply views/show.html]: it shows the date in the screenshot!

File: views/summary.html (hunk 4 above)
Comment (mathewantony31) [reply views/summary.html]: haha no i didn't. thank you for catching this

File: views/show.html (hunk 3 above)
Comment (mathewantony31) [reply views/show.html]: <3 these suggestions, updating

**Pull request merged**